src/main/scala/edu/ucla/cs/bwaspark/Usage.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package edu.ucla.cs.bwaspark
+
+object Usage {
+  val usage: String = "Usage 1: upload raw FASTQ file(s) to HDFS\n" +
+    "Usage: upload-fastq [-bn INT] isPairEnd filePartitionNum inputFASTQFilePath1 [inputFASTQFilePath2] outFileHDFSPath\n\n" +
+    "Required arguments (in the following order): \n" +
+    "isPairEnd: pair-end (1) or single-end (0) data\n" +
+    "inputFASTQFilePath1: the first input path of the FASTQ file in the local file system (for both single-end and pair-end)\n" +
+    "inputFASTQFilePath2: (optional) the second input path of the FASTQ file in the local file system (for pair-end)\n" +
+    "outFileHDFSPath: the root path of the output FASTQ files in HDFS\n\n" +
+    "Optional arguments: \n" +
+    "-bn (optional): the number of lines to be read in one group (batch)\n\n\n" +
+    "Usage 2: use CS-BWAMEM aligner\n" +
+    "Usage: cs-bwamem [-bfn INT] [-bPSW (0/1)] [-sbatch INT] [-bPSWJNI (0/1)] [-jniPath STRING] [-oType (0/1/2)] [-oPath STRING] [-localRef INT] [-R STRING] [-isSWExtBatched (0/1)] [-bSWExtSize INT] [-FPGAAcc (0/1)] isPairEnd fastaInputPath fastqHDFSInputPath\n\n" +
```

Comment:
where is this class used?  If still used, the command name and arguments need to be updated to match those in `CloudScaleBWAMEM`